### PR TITLE
Improve homepage

### LINKED ISSUE #65: Improve homepage
- increase `widgets-container` width to 80%
- Improve font-spacing
- Add respactive links to `widget-title` i.e. `/events/` for events widget-title.

### PR DESCRIPTION
**Pull request by mohitkyadav**
Fixes #65 
Letter-spacing, Font-size and widgets-container width increased, Added links for `widgets-title`.
Deployed here
http://13.58.190.145:8000/
admin - test
password - test

### CODE REVIEW

**Comment by mohitkyadav** [comment]
screenshot 77

File: wsgi/odyssy/basic/templates/basic/index.html [hunk 1]
```diff
@@ -36,17 +36,17 @@
             <!-- Announcements -->
             <div class="col-sm-4" id="announcements">
                 <div class='announcement-content '>
-                    <div class="widgets-title"><h2>Announcements</h2></div>
+                    <div class="widgets-title"><h2><a href="/announcement/">Announcements</a></h2></div>
```
Comment (Monal5031) [inline wsgi/odyssy/basic/templates/basic/index.html]: Don't use static urls.

**Comment by singhpratyush** [comment]
@mohitkyadav: What's with the colour in the event section?

File: wsgi/odyssy/basic/templates/basic/index.html (hunk 1 above)
Comment (mohitkyadav) [inline wsgi/odyssy/basic/templates/basic/index.html]: @Monal5031 I wouldn't, but there's no view created for all announcements, Is there?

**Comment by mohitkyadav** [comment]
@singhpratyush sir this is event section,
screenshot 79

and this is on hovering curser
screenshot 80

File: wsgi/odyssy/basic/templates/basic/index.html (hunk 1 above)
Comment (Monal5031) [inline wsgi/odyssy/basic/templates/basic/index.html]: https://github.com/iiitv/Odyssy/blob/master/wsgi/odyssy/announcement/urls.py
here it is mentioned `name='index-view'` you can use that.

Comment (mohitkyadav) [inline wsgi/odyssy/basic/templates/basic/index.html]: @Monal5031 the `index-view` you are talking about is view for the homepage and I need a view for all announcements, like `event-view`.

Comment (Monal5031) [inline wsgi/odyssy/basic/templates/basic/index.html]: @mohitkyadav yes there is a view which you are talking about but that html file is named as `index.html` in announcement app and its name as `index-view` unlike other apps, where they are named as `event_list.html` or `news_list.html` and have a name similar to this.

**Comment by mohitkyadav** [comment]
@singhpratyush is this allright sir?
screenshot 82

**Comment by singhpratyush** [comment]
@mohitkyadav: Please deploy on cloud and share link.

**Comment by mohitkyadav** [comment]
@singhpratyush 
Deployed here
http://13.58.190.145:8000/
admin - test
password - test

**Comment by mohitkyadav** [comment]
If there's one photo at a time, I thnk it will cover almost whole screen...

**Comment by yashLadha** [comment]
Yaa I also thinks so one photo would be better. Go check for Bootstrap
Carousal . I think that would help.

On Sat, Jun 17, 2017 at 7:07 AM, Mohit Kumar Yadav <notifications@github.com

**Comment by yashLadha** [comment]
[http://getbootstrap.com/javascript/#carousel](http://getbootstrap.com/javascript/#carousel)

**Comment by geekyJock8** [comment]
@mohitkyadav  @Prakash2403  I also think that one image at a time will be better. If you want to avoid the image covering the whole screen, maybe you can try something like this:
image

Also, I feel that the original font spacing and size on http://www.iiitvadodara.ac.in/ looks better.

**Comment by mohitkyadav** [comment]
Should I change carousal in same PR?

**Comment by yashLadha** [comment]
GFI @mohitkyadav

**Comment by Monal5031** [comment]
@yashLadha @Prakash2403 @geekyJock8 sir I think this should go in #13 as the whole carousel will be updated in it, so there is no point in making temporary changes in this PR, your views?

**Comment by yashLadha** [comment]
Its ok them @Monal5031 .

**Comment by yashLadha** [comment]
Looking Good . Can u please rebase al the commits. @mohitkyadav

**Pull request closed without merging**